README.md

```diff
@@ -74,7 +74,7 @@ r = RotMatrix(aa)
 ```julia
 using Rotations, StaticArrays
 
-# create the null rotation (identity matrix)
+# create the identity rotation (identity matrix)
```

Comment:
I'm not familiar with wordings, but the word "null rotation" seems different from "identity rotation".
* https://www.gregegan.net/SCIENCE/GR2plus1/NullRotations.html
* https://arxiv.org/abs/hep-th/0203201
* https://terrytao.wordpress.com/tag/null-rotations/